### PR TITLE
Gives the chainsaw start/stop/loop SFX, makes the arm-mounted chainsaw a subtype of a regular chainsaw

### PR DESCRIPTION
**Pull request by Rhials**
## About The Pull Request

The chainsaw now has an audio loop, as well as SFX for starting and stopping. I'm pretty confident that I did the attribution stuff correctly.

https://user-images.githubusercontent.com/28870487/225054900-4bf87ee7-9245-4004-b414-77d0e6e83a7c.mp4

I think this came out well. It may be a bit loud, but I wanted the volume to be about the same as how the chainsaw slash sound comes out. Audio stuff isn't really my forte, so let me know if anything sounds off, or if any adjustments should be made.

To make this work for the mounted chainsaw (more specifically, the abstract object that enters your hand), it has been made into a chainsaw subtype. An updatepaths script has been included in the event that someone has a map where the former path was included (why??).
## Why It's Good For The Game

Makes the chainsaw more menacing. It's one of the most powerful and beloved pieces of crew weaponry, in addition to being a fucking chainsaw. If you aren't keen on walking around the hallways with the chainsaw loop playing like the badass you are, you can always just keep it off until you get into a fight.

Gives an extra bit of immersion for those special moments where you get to chainsaw someone to death.
## Changelog
:cl:
soundadd: The chainsaw now has SFX for starting, stopping, and idling.
/:cl:

### CODE REVIEW

File: sound/attributions.txt [hunk 1]
```diff
@@ -37,3 +37,12 @@ alien_explode.ogg is adapted from zimbot's Frozen Lettuce upload https://freesou
 alien_organ_cut.ogg is adapted from some CC 0 flesh sound, lesaucisson's swoosh https://freesound.org/people/lesaucisson/sounds/585256/, and a snippet of EminYILDIRIM's tree cutting clip https://freesound.org/people/EminYILDIRIM/sounds/608343/
 
 standard_stamp.ogg is adapted from tom_woysky's "Stamp.wav" https://freesound.org/people/tom_woysky/sounds/348316/
+
+chainsaw_loop.ogg is adapted from Audionautics "Chainsaw Idle Loops"
```
Comment (san7890) [inline sound/attributions.txt:41]:
```suggestion
chainsaw_loop.ogg is adapted from Audionautics "Chainsaw Idle Loops", which is licensed under CC Attribution 3.0
```

File: sound/attributions.txt [hunk 2]
```diff
@@ -37,3 +37,12 @@ alien_explode.ogg is adapted from zimbot's Frozen Lettuce upload https://freesou
...
+https://freesound.org/people/Audionautics/sounds/171652/
+chainsaw_start.ogg is adapted from 8 Chainsaw Pulls.wav by Stefan021
```
Comment (san7890) [inline sound/attributions.txt:43]:
```suggestion
chainsaw_start.ogg is adapted from 8 Chainsaw Pulls.wav by Stefan021 (CC 0)
```

File: sound/attributions.txt [hunk 3]
```diff
@@ -37,3 +37,12 @@ alien_explode.ogg is adapted from zimbot's Frozen Lettuce upload https://freesou
...
+https://freesound.org/s/431740/
+chainsaw_stop.ogg is adapted from kyles "chainsaw sawing short cuts +stop.flac"
```
Comment (san7890) [inline sound/attributions.txt:45]:
```suggestion
chainsaw_stop.ogg is adapted from kyles "chainsaw sawing short cuts +stop.flac" (CC 0)
```

**Review by san7890 — CHANGES_REQUESTED** [review]
even though some of the other links here don't have the exact license, let's just do it like this in case freesound.org implodes or something and we lose all track of the artist's intent

**Comment by nukularpower** [comment]
its been annoying me forever that chainsaws on /tg have no sound, very good!

**Comment by Livrah** [comment]
House Edge should use this too.

**Pull request merged**